README.md

```diff
@@ -203,6 +203,12 @@ zip_stream_close(zip);
 free(buf);
 ```
 
+* Extract an in-memory zip archive to a directory.
+
+```c
+int ziperr = zip_stream_extract(zipstream, zipstreamsize, outdir, NULL, NULL);
+```
+
```

Comment:
Can you put this example close to the section _Extract a zip archive into a folder_?
Just mention about in memory variant.

Reply:
Will do!